### PR TITLE
Reduce probability of horizontal scrolling on resource lists

### LINKED ISSUE #1146: Reduce probability of horizontal scrolling on resource lists
## Description
Currenlty, in all resource lists except RL, we have a couple of buttons on the right side of a list item that allow us to edit/view/... an item. These icons make it more probable that a resource list would have to be horizontally scrolled. In cases described below, extract some icons into a 'more menu' 3-dot icon and in places where we have an edit icon only, ensure the rows of data can instead be clickable. 

## Current behaviour

Image

## Expected behaviour

Image

## Implementation list

<details>

<summary>Compose</summary>

- [x] Charts
- [x] Add Delete to Charts
- [x] Modules
- [x] Add Delete to Modules
- [x] Namespace
- [x] Add Delete to Namespace
- [x] Update Namespace Edit to stop using $store.dispatch

</details>

<details>

<summary>Admin</summary>

***For the Admin webapp the task would be to make sure the resource lists used no longer have an edit button. Each row should be clickable***

Image

- [x] APIGW
- [x] Connections
- [x] Federated Nodes *pending locale*
- [x] Sessions
- [x] Workflow
- [x] Application
- [x] AuthClient ([Address Inability to delete authclients](https://github.com/cortezaproject/corteza/issues/1216))
- [x] Queue
- [x] Role
- [x] Senstivity Level
- [x] Template
- [x] Users

</details>

<details>

<summary>Reporter</summary>

- [x] List of Reports

</details>

<details>

<summary>Workflow</summary>

- [x] Workflow List

</details>

### PR DESCRIPTION
**Pull request by EmmyMay**
# The following changes are implemented
TODO: Summary

# Changes in the user interface:
TODO: Add screenshots, recordings or remove this section

# Checklist when submitting a final (!draft) PR
 - [ ] Commits are tidied up, squashed if needed and follow guidelines in CONTRIBUTING.md
 - [ ] Code builds
 - [ ] All existing tests pass
 - [ ] All new critical code is covered by tests
 - [ ] PR is linked to the relevant issue(s)
 - [ ] Rebased with the target branch

### CODE REVIEW

**Review by Fajfa — CHANGES_REQUESTED** [review]
1. Revert editors
2. Add reusable handle delete to listHelpers that takes resource and deletes it
3. Try adding one style to CResourceList that covers actions
4. Remove everything except handle row clicked, clickable and slot definition (delete,permissions)
5. Make sure slot definition permissions are correctly applied

File: client/web/admin/src/components/Apigw/CRouteList.vue [hunk 1]
```diff
@@ -58,13 +61,14 @@
...
-          button-variant="light"
-          class="ml-1"
+          button-variant="link text-decoration-none text-dark regular-font rounded-0"
+          class="text-dark d-print-none border-0"
         >
           <font-awesome-icon :icon="['fas', 'lock']" />
           {{ $t('permissions') }}
         </c-permissions-button>
 
+        <!-- v-if false? Please take a look at this-->
```
Comment (Fajfa) [inline client/web/admin/src/components/Apigw/CRouteList.vue]: Up to you, you can keep it or remove it

**Review by Fajfa — CHANGES_REQUESTED** [review]
1. Remove refs from resource lists
2. Remove unused classes from Lists
3. use filterList when deleting resource
4. Rename handleListDelete to for example `handleItemDelete`
5. Move the additional actions to separate branch for now

**Review by Fajfa — APPROVED** [review]

**Comment by katrinDY** [comment]
Issue found after QC:
- compose -> module list: can't access related pages and 3 dots

https://github.com/cortezaproject/corteza/assets/85161724/69473e65-1278-4ee7-be0e-d9365811026d

- admin -> integration gwt -> routes: style issue on 3 dots menu open

https://github.com/cortezaproject/corteza/assets/85161724/69b7bbff-5967-478f-b1d4-26bc87009015

**Comment by katrinDY** [comment]
Final batch 🤞:
- 3 dots/related pages btns not visible from mobile device in all webapps
- wf list and admin wf list: the permission to delete a wf enables/disables the permission to undelete a wf. For example, if I set canDelete to true I'm able to delete and undelete a wf but if I set it to true I cannot delete and undelete a wf

**Review by Fajfa — APPROVED** [review]

**Review by Fajfa — APPROVED** [review]

**Pull request merged**